isis/src/qisis/objs/Footprint2DView/Footprint2DView.cpp

```diff
@@ -313,39 +316,21 @@ namespace Isis {
     }
   }
 
-
-  /**
-   * Returns a list of actions for the permanent tool bar.
-   *
-   * @return @b QList<QAction*> The actions
-   */
-  QList<QAction *> Footprint2DView::permToolBarActions() {
-    return m_permToolBar->actions();
-  }
-
-
-  /**
-   * Returns a list of actions for the active tool bar.
-   *
-   * @return @b QList<QAction*> The actions
-   */
-  QList<QAction *> Footprint2DView::activeToolBarActions() {
-    QList<QAction *> actions;
-    actions.append(m_activeToolBarAction);
-    return actions;
-  }
-
-
   /**
-   * Returns a list of actions for the tool pad.
-   *
-   * @return @b QList<QAction*> The actions
+   * A slot function that is called when directory emits a siganl that an active
+   * control network is set. It enables the control network editor tool in the
+   * toolpad and loads the network.
+   * TODO 2018-06-14 Kaitlyn Lee - Commented out code that loads the network,
+   * since it does not actually load the network. Left it here to research further.
    */
-  QList<QAction *> Footprint2DView::toolPadActions() {
-    return m_toolPad->actions();
+  void Footprint2DView::enableControlNetTool(bool value) {
+    m_controlNetTool->setEnabled(value);
+    // if (value) {
```

Comment:
Can this be deleted?

Reply:
I left a comment above just saying I left it in there to remind myself how the behavior works and I will research it further once Tracie tells me how it should be done.

Reply:
Do you have this version built somewhere? I would like to test since I can't see anywhere the loadNetwork called which is the only way the network is loaded into footprint view.  Am I missing a call or connection somewhere?

Reply:
I think leaving the call to ControlNetTool::loadNetwork in Footprint2DView::enableControlNetTool would be the cleanest.

Reply:
I have a build in /work/projects/IPCE/m05440_m05389/isis/ Calling loadNetwork() here does not actually load the network. There is a method, displayNetwork(), that displays the network if the "Display" button is clicked in the active toolbar of the view. I was under the impression that Ken told us to keep the functionality of qmos, where a network is not loaded until you click on the CnetEditor tool.

Reply:
Ok, I see how it is working. Thinking we might want to document this somewhere:
I think MosaicControlNetTool::displayNetwork() only turn the display of pt on and off. It assumes the network has already been set and the GraphicsItems created.  But, MosaicControlNetTool::updateTool call ::openControlNet which then calls ::loadNetwork.  I think ::updateTool is called when tool is enabled or chosen.  So, that is why we don't need to call loadNetwork.